build/Defaults/Desktop/app.config

```diff
@@ -19,7 +18,51 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
-        <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="1.2.0.0"/>
+        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0"/>
+      </dependentAssembly>
+      <dependentAssembly>
```

Comment:
Plus there are other issues with the current implementation of auto-generating BRs. I tried and failed.